docs/en/pact-properties-api.md

```diff
@@ -944,6 +944,20 @@ Add seconds to a time
 
 Supported in either invariants or properties.
 
+### diff-time {#FTemporalDiff}
+
+```lisp
+(diff-time a b)
+```
+
+* takes `a`: `time`
+* takes `b`: `time`
+* produces `decimal`
+
+Time difference in seconds
```

Comment:
I think it's worth spelling out explicitly that it's `a - b` — that is, the first argument is the minuend and the second one is the subtrahend.

Reply:
the `docs/en/*/md` files are auto-generated by some tests.